addons/account/models/account_move.py

```diff
@@ -1660,6 +1662,11 @@ def _compute_amount_total_words(self):
         for move in self:
             move.amount_total_words = move.currency_id.amount_to_text(move.amount_total).replace(',', '')
 
+    @api.depends('move_type')
+    def _compute_incoterm(self):
+        for move in self:
+            move.invoice_incoterm_id = move.company_id.incoterm_id if move.move_type.startswith('out_') else False
```

Comment:
```suggestion
        for move in self.filtered(lambda m: m.move_type.startswith('out_'):
            move.invoice_incoterm_id = move.company_id.incoterm_id
```

Reply:
> And I don't know if we want to reset the incoterm for in_invoice if you change the company. Debatable

True, I'll change to `move.company_id.incoterm_id if move.move_type.startswith('out_') else move.invoice_incoterm_id`